src/config/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright 2019-present MongoDB Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project(config VERSION 0.0.1 LANGUAGES CXX)
+
+CreateGennyTargets(
+    NAME    config
+    TYPE    STATIC
+    DEPENDS
+        MongoCxx::bsoncxx
```

Comment:
unused now?

Reply:
Ya nice catch - I don't think `config` needs to exist as a separate cmake project to just moved `Node.hpp` and friends into `gennylib`.